src/components/hoc/withAdapter.jsx

```diff
@@ -87,7 +86,7 @@ export default function withAdapter(WrappedComponent, adapterFactory) {
        * @returns {Promise} A promise that resolves when the adapter finishes disconnecting
        */
       async function disconnect() {
-        setAdapterConnected(false);
+        global.testRendererAct(() => setAdapterConnected(false));
```

Comment:
is this needed? the act() needs to be called after an await, but here it seems that setAdapterConnected() is called synchronously.

Reply:
You are right, I'll remove it.